src/FancyTestdoxPrinter.php

```diff
@@ -89,6 +92,9 @@ public function startTest(Test $test)
         } elseif ($test instanceof PhptTestCase) {
             $className  = $class;
             $testMethod = $test->getName();
+        } elseif ($test instanceof TestSuite) {
+            $className = $class;
+            $testMethod = "setUpBeforeClass";
```

Comment:
Can this never be any other method? Seems tricky to hardcode this.

Reply:
This problem only occurs when there are errors in static functions I suspect. I will double check this.